src/main/java/com/google/cloud/tools/appengine/api/devserver/DefaultRunConfiguration.java

```diff
@@ -51,16 +50,6 @@
   private File datastorePath;
 
   @Override
-  public List<File> getAppYamls() {
-    return appYamls;
-  }
-
-  public void setAppYamls(List<File> appYamls) {
-    this.appYamls = appYamls;
-  }
-
-
-  @Override
   public List<File> getServices() {
```

Comment:
There is some javadoc on the method in `RunConfiguration`. Since devappserver doesn't support flex, and local runs are executed from the build directory (not stage where app.yamls are), I think the javadoc is correct, but feel free to correct me.